lib/spotify_browser.dart

```diff
@@ -5,11 +5,13 @@ library spotify;
 
 import 'dart:async';
 import 'dart:convert';
+import 'dart:html';
 
 import 'package:http/browser_client.dart' as http;
+import 'package:http/http.dart' as http_all;
```

Comment:
Will this still work in the browser?

Reply:
I have checked o both android and web and it worked. but actually it may be cleaner to turn it to:
```dart
import 'package:http/browser_client.dart' as http;
import 'package:http/src/base_client.dart';
import 'package:http/src/request.dart';
import 'package:http/src/response.dart';
```
what do you think?

Reply:
I guess tree shaking gets rid of the stuff not being used, but importing explicitly might be better, yes!

Reply:
fixed